provider.go

```diff
@@ -13,8 +15,22 @@ import (
 
 	"github.com/hashicorp/terraform/helper/schema"
 	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/coreos/fleet/version"
+	"github.com/coreos/fleet/registry"
+
+	etcd "github.com/coreos/fleet/Godeps/_workspace/src/github.com/coreos/etcd/client"
 )
 
+const oldVersionWarning = `####################################################################
+WARNING: fleetctl (%s) is older than the latest registered
+version of fleet found in the cluster (%s). You are strongly
+recommended to upgrade fleetctl to prevent incompatibility issues.
```

Comment:
Could this message be clarified to make it more clear that it's not the `fleetctl` binary that needs to be updated, but the version of fleet that `terraform-provider-fleet` is linked against?